### ISSUE TITLE
Server fails when favicon.ico is requested by a browser

### ISSUE THREAD

**Issue by andrewbt** [CityOfPhiladelphia/soda-carto#24]
We should probably make things more fault-tolerant

**Comment by timwis** [comment]
Yeah, I don't know why the browser is requesting `favicon.ico` on a json page -- normally it doesn't do that if the response `Content-Type` header is set to `application/json`. In fact, I thought I added that header to `server.js` and that fixed it. But it's definitely happening in the lambda function :/ at least it doesn't fail, it just shows an error in the network tab.

Are you just using `server.js` because it's easier to test? If so, we can definitely make it handle unknown routes pretty easily. I was also thinking of having `server.js` just delegate processing to `handler.js` (since `handler.js` is just a pure function) to have more parity.
